paper/paper.md

```diff
@@ -60,6 +60,8 @@ affiliations:
     index: 8
   - name: Barcelona Supercomputing Center (BSC), Plaça Eusebi Güell, 1-3 Barcelona, Spain
     index: 9
+  - name: Department of Informatics, University of Bergen, Norway
+    index: 10
```

Comment:
Is Hans' affiliation still valid? Or should it be KU now?
And Jennifer's does still need to be added, right?

Reply:
@hansioan @jrbjensen